bitnami/kubernetes-event-exporter/templates/deployment.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
     {{- end }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  {{- if .Values.revisionHistoryLimit }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  {{- end }}
```

Comment:
Is that if-block really needed?

Reply:
Hi @fevisera, 
thanks for your review. :-)

You are right. It isn't really needed for if the value is not present it would fallback to the default of 10. Only if one wants to overwrite this default behavior, the value must be set.

So i will remove the if-block.